src/lib/isMobilePhone.js

```diff
@@ -77,6 +77,7 @@ const phones = {
   'fr-FR': /^(\+?33|0)[67]\d{8}$/,
   'fr-GF': /^(\+?594|0|00594)[67]\d{8}$/,
   'fr-GP': /^(\+?590|0|00590)[67]\d{8}$/,
+  'fr-HT': /^(\(?\+?509\)?)[0-9]{8}$|^(\(?\+?509\)?) [0-9]{2} [0-9]{2} [0-9]{4}$/,
```

Comment:
Can you remove spaces and parentheses from regex? Their purpose is only asthetic and shouldn't be counted in the validation. 
Also, looks like landline numbers are passing, can you change your regex in order to validate only mobile phone numbers (Source: [Wikipedia](https://en.wikipedia.org/wiki/Telephone_numbers_in_Haiti) and [Gov website](http://www.conatel.gouv.ht/sites/default/files/PNN2014.pdf))